README.md

```diff
@@ -27,6 +27,9 @@
     <a href="https://greptime.com/slack"><img src="https://img.shields.io/badge/slack-GreptimeDB-0abd59?logo=slack" alt="slack" /></a>
 </p>
 
+> [!WARNING]  
+> Our default branch has changed from `develop` to main. Please update your local repository to use the `main` branch.
+
```

Comment:
You could merge the `main` branch instead of the `develop` branch. You could rename the `develop` branch to `main` in your forked repo via GitHub UI.

Reply:
done, I submit a new PR:
https://github.com/GreptimeTeam/greptimedb/pull/3031